renderer/src/main.cpp

```diff
@@ -24,9 +24,13 @@ int main(int argc, char * argv[]) {
 
         auto renderer_conf = rcbe::utils::readFromFile<rcbe::rendering::RendererConfig>("datamodel/data/rendering/default_renderer_config.json");
 
-        rcbe::rendering::GLRenderer renderer { std::move(renderer_conf), window };
+        window->map_window();
 
-        auto renderer_handle = renderer.start_async();
+        auto renderer = rcbe::rendering::make_renderer_ptr(std::move(renderer_conf), window->get_context());
+
+        window->set_renderer(renderer);
```

Comment:
This one should talke `unique_ptr`

Reply:
It will in time, but now it would be too incinvinient

Reply:
Try to implement it now, if too bothersome, we will leave it for later